components/script/dom/gpudevice.rs

```diff
@@ -230,22 +249,19 @@ impl GPUDevice {
                 bgls.push(webgpu::WebGPUBindGroupLayout(bgl));
```

Comment:
Seems like we no longer use the `bgls`  vector in line 254 - we push elements into it, but the vector is not returned. Can we simply remove it?

Reply:
sure, can we also remove `let mut bgls = Vec::with_capacity(max_bind_grps as usize);` in line 245?

Reply:
Right, that is the one I was referring to, just mixed up the line number.

Reply:
Done 👍